api/api.yaml

```diff
@@ -861,11 +859,11 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
-  /v2/identities/{identifier}/credentials/{id}/qrcode:
+  /v2/identities/{identifier}/credentials/{id}/offer:
     get:
-      summary: Get Credentials QR code
-      operationId: GetCredentialQrCode
-      description: Returns a a json that can be used to create the QR Code to scan for accepting a claim. |
+      summary: Get Credentials Offer
+      operationId: GetCredentialOffer
+      description: Returns universal/deep link to offer or raw offer message can be used for accepting a claim. |
```

Comment:
Returns **a** universal/deep-link to **an** offer or raw offer message **that** can be used for accepting a claim.